### PR TITLE
Agile and DevOps Brussels blogpost - first version

### PR DESCRIPTION
**Pull request by wnivelle**

### CODE REVIEW

File: _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+### Agile is (not) only for IT
+
+The talk started with explaining that all [12 principles of the Agile Manifesto](http://agilemanifesto.org/principles.html){:target="_blank"} can be applied to all layers of the organization, it's not only for the IT department (hence the title). The Agile Manifesto enables everyone to focus on the same objective and towards the same goals.
+
+In order to get everyone to focus, it's important to know in what situation you are. To find out, you can use the [Cynefin framework](https://hbr.org/2007/11/a-leaders-framework-for-decision-making){:target="_blank"}. This framework explains where you belong by differentiating between 4 different domains. 
+
+<img alt="The different domains of the Cynefin framework" src="{{ '/img/agile-devops-brussels-2018/cynefin.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 800px;">
+
+* **Complex**: non-lineair, never complete.
+* **Complicated**: analysis is needed,  a lot of possibilities, multiple possible answers.
```
Comment (Deryaa) [inline _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md]: I think there are two spaces after the first comma?

File: _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+* **Chaotic**: take action, dynamic, nothing is the same.
+* **Obvious**: predictable and repeatable, one or few answers.
+
+To take a real example: 'Planning a birthday party for your child':
+* If you leave the children all alone the entire afternoon, you have no idea what happens during your absence or what the state of your house will be when you return: **Chaos**.
+* If you plan the entire afternoon, with a timetable, to ensure that everything happens as planned: **Complicated**.
+* If the children are obedient and don't do anything wrong at all (not exactly realistic): **Simple**.
+* If you set boundaries, rules, adjust continously: **Complex**.
```
Comment (Deryaa) [inline _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md]: Continuously

File: _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+When you're in a complex situation, you can apply the agile principles.
+
+The other part of the talk was about the Agile Wave and how it affects other parts of your organization other than IT. If the IT department starts working according to the agile principles, there is always an impact on the other domains of your organization.
+For instance, if you have a team that goes agile, there's an impact on items like budget, bonusses, costs, collaboration, estimation and so on. So the other domains need to adapt as well.
```
Comment (Deryaa) [inline _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md]: bonuses

File: _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+
+
+### Agile Maturity
+
+The next talk's subject was the issue of measuring agile maturity in an organization. 
+Agile maturity can be answered with 3 questions:
+* Why am I doing it?
+* What am I doing?
+* I am convinced about it's outcome.
```
Comment (Deryaa) [inline _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md]: its

Comment (Deryaa) [reply _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md]: Maybe formulate the third one also as a question?

File: _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+    - Obviously, there's a direct correlation between badged hours and productivity.
+* **Do DevOps with Ops**
+    - It's a fancy name.
+    - Infrastructure should still be separate so it can be controlled.
+* **Create DevOps department to coordinate between Dev and Ops**
+    - More control and coordination is better
+* **Ban teams from using their own tools**
+    - Management knows best what tools their teams need, all teams need the exact same things.
+
+*To be clear, this list contains a fair bit of sarcasm.*
```
Comment (Deryaa) [inline _posts/2018-07-02-Agile-DevOps-Summit-Brussels.md]: Hooray!

**Comment by wnivelle** [comment]
Requested changes have been made.

**Pull request merged**